go.mod

```diff
@@ -0,0 +1,49 @@
+module github.com/topfreegames/khan
+
+go 1.15
```

Comment:
Is this the correct go version?

Reply:
No, this the version that I currently use local, I will fix this